lib/posix-process/process.h

```diff
@@ -41,6 +41,21 @@
 
 #define TIDMAP_SIZE (CONFIG_LIBPOSIX_PROCESS_MAX_PID + 1)
 
+/* Notice: The RUNNING state is not necessarily in sync with the state
+ * of the undelrying uk_thread (may be blocked by the scheduler).
```

Comment:
s/undelrying/underlying/
Btw, you also used both `uk_thread` and `uk-thread` in this paragraph.